Analysis/Utkscan/core/source/TimingCalibrator.cpp

```diff
@@ -32,7 +32,7 @@ TimingCalibrator* TimingCalibrator::get() {
 }
 
 void TimingCalibrator::ReadTimingCalXml() {
```

Comment:
I updated this to follow the same programming as in Notebook. This class needs to be updated to handle the geometric correction a little differently, but this will bring it in line with the scope of this pull request.